rasa/nlu/constants.py

```diff
@@ -1,9 +1,13 @@
 TEXT = "text"
 INTENT = "intent"
 RESPONSE = "response"
+ACTION_TEXT = "action_text"
 
 RESPONSE_KEY_ATTRIBUTE = "response_key"
 
+MESSAGE_ACTION_NAME = "action_name"
+MESSAGE_INTENT_NAME = "intent_name"
```

Comment:
to be consistent with the other constants:
```suggestion
ACTION_NAME = "action_name"
INTENT_NAME = "intent_name"
```

Reply:
Done in the next PR

Reply:
why in the next PR? let's not mix PRs